samer/assets/img/test.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<!-- saved from url=(0037)http://127.0.0.1:5500/test.html#img-1 -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
+    <title>test</title>
+<link rel="stylesheet" href="./test_files/style.css"></head>
+<body>
+    <div class="slide-container">
+        <span id="img-1"></span>
+        <span id="img-2"></span>
+        <span id="img-3"></span>
+
+        <div class="img-container">
+            <img class="gal" src="./test_files/above.jpg" alt="Aleppo" title="Aleppo" width="800px" height="600px">
+            <img class="gal" src="./test_files/above1.jpg" alt="Aleppo" title="Aleppo" width="800px" height="600px">
+            <img class="gal" src="./test_files/above2.jpg" alt="Aleppo" title="Aleppo" width="800px" height="600px">
+        </div>
+        <div class="button-cont">
+            <a href="http://127.0.0.1:5500/test.html#img-1" class="slider-button"></a>
+            <a href="http://127.0.0.1:5500/test.html#img-2" class="slider-button"></a>
+            <a href="http://127.0.0.1:5500/test.html#img-3" class="slider-button"></a>
+        </div>
+    </div>
+<!-- Code injected by live-server -->
+<script type="text/javascript">
```

Comment:
what is this script about? are you doing JS in private? =)

Reply:
I mean what is this file about in general? an test.html inside your image folder?